src/Example/Views/DataGridView.axaml

```diff
@@ -1,108 +1,107 @@
-<Window xmlns="https://github.com/avaloniaui"
-        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-        xmlns:views="clr-namespace:Example.Views"
-        x:Class="Example.Views.DataGridView">
-    <Grid ColumnDefinitions="* 35">
-        <Grid>
-            <DataGrid ItemsSource="{Binding}"
-                      Name="Grid"
-                      SelectionMode="Single"
-                      Focusable="True"
-                      AutoGenerateColumns="False"
-                      HorizontalScrollBarVisibility="Disabled">
-                <DataGrid.Columns>
-                    <DataGridTextColumn Header="Id"
-                                        Binding="{Binding Id}"
-                                        MaxWidth="5"
-                                        Width="5" />
-                    <DataGridTextColumn Header="Title"
-                                        Binding="{Binding Title}"
-                                        Width="*" />
-                </DataGrid.Columns>
-            </DataGrid>
-            <TextBlock Text="movies"
-                       VerticalAlignment="Top"
-                       HorizontalAlignment="Left"
-                       Margin="4,0,0,0" />
-        </Grid>
-        <Grid Grid.Column="1">
-            <Border BorderThickness="1"
-                    BorderBrush="DarkGray"
-                    DataContext="{Binding ElementName=Grid, Path=SelectedItem}">
-                <Grid ColumnDefinitions="Auto *"
-                      x:DataType="views:TheItem">
-                    <Grid.RowDefinitions>
-                        <RowDefinition Height="Auto" />
-                        <RowDefinition Height="Auto" />
-                        <RowDefinition Height="Auto" />
-                        <RowDefinition Height="Auto" />
-                        <RowDefinition Height="Auto" />
-                        <RowDefinition />
-                    </Grid.RowDefinitions>
-                    <StackPanel Orientation="Vertical"
-                                Margin="1,0">
-                        <StackPanel.Styles>
-                            <Style Selector="TextBlock">
-                                <Setter Property="Foreground"
-                                        Value="Gray" />
-                            </Style>
-                        </StackPanel.Styles>
-                        <TextBlock Text="Id" />
-                        <TextBlock Text="Title" />
-                        <TextBlock Text="Genre(ALT+DOWN)" />
+<Grid xmlns="https://github.com/avaloniaui"
```

Comment:
If you are making any other changes to this PR. Lets drop this project, it's nonsense now.